WcaOnRails/app/models/country.rb

```diff
@@ -38,6 +47,6 @@ def self.find_by_iso2(iso2)
       # "Tunisia" even if the country displayed is actually "Tunisie"...)
       [e.name_in(l), e.id]
```

Comment:
> Can we use locale and country instead of l and e?
Yes, I can do even do it in this PR.

> Isn't "search will behave weirdly" a euphemism for "search will no longer work"?
Well it *will* work, but it will only search for English names.
There is definitely room for improvement here, and an information tooltip for the user would definitely be necessary!